### ISSUE TITLE
Windows installer for v1.2

### ISSUE THREAD

**Issue by achilleas-k** [G-Node/nixpy#202]

**Comment by matham** [comment]
Hey, I have made a lot of progress automotaing the process of creating nixpy (nixio?) wheels for python on appveyor. These wheels would include all the nix binaries so it would just be `pip install wheel_name` and everything would just work (without having to install nix first).

I'm using the last nix release for windows, but unfortunately, because of the `deleteDimension` change (https://github.com/G-Node/nix/commit/24a214a9cd63eb73a14b5cc908554cfc6b080c11), current nixpy master which relies on this change is out of date of the last nix release, which is absent that change. So the compilation fails: (https://ci.appveyor.com/project/matham/nixpy). Would it be possible to create a new nix release? I didn't fancy compiling nix master on appveyor ;)

Also, a lot of the difficulty was getting python to not use visual studio 2008 rather than 2013. Appveyor comes preinstalled with all visual studio versions so python kept falling back to 2008 no matter what I did. Removing 2008 from the registry did the trick. But, I'm still looking for a way to force python to not use 2008 with 2.7 without having to mess with the registry. Maybe using `DISUTILS_USE_SDK`.
